.gitignore

```diff
@@ -100,3 +100,8 @@ docs/*.html
 # IDE related
 .idea/
 .vscode/
+
```

Comment:
Can you remove these changes to `.gitignore`? It could make sense to add these but it's not related to this PR.

Reply:
Will do. Will have a think about it and might make a separate PR if it still makes sense on second thought.